woocommerce/woocommerce-gutenberg-products-block
# Update NPM dependencies and script dependency handling

Updates `package.json` to use latest NPM packages, and latest WordPress dependencies. Also uses latest `serialize-javascript` to fix a reported vulnerability.

With the update to `@wordpress/scripts`, the way in which dependencies are generated has changed from `json` files to `php` files with arrays of data. I've updated our asset handlers to match.

### How to test the changes in this Pull Request:

1. Successful `npm install` and `npm run build`
2. Successful `npm run test`
3. Spot check existing blocks for errors.
